services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/service/DataPlaneServiceNew.java

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+package com.futurewei.alcor.dataplane.service;
+
+import com.futurewei.alcor.web.entity.dataplane.InternalDPMResultList;
+import com.futurewei.alcor.web.entity.dataplane.refactor.NetworkConfiguration;
+
+public interface DataPlaneServiceNew {
```

Comment:
Which interface is in use? DataPlaneService or DataPlaneServiceNew :-)

Reply:
DataPlaneService is in use now.